blog/2022/12-16-resoto-aws-permissions-rollout/index.md

```diff
@@ -0,0 +1,123 @@
+
+This post assumes that your organization is already set up with an AWS Organization and that it meets all [the prerequisites required to use StackSets in your organization](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs.html).
+
+### Deploying the StackSet from the AWS Console
+
+#### Finding the Organization Unit (OU)
+
+Find out the Organization Unit (OU) in your organization where you want to deploy the `ResotoAccess` role. The role will be deployed to all accounts in this OU and all child OUs.
+
+Navigate to the Organization Home in the [AWS Organizations Console](https://console.aws.amazon.com/organizations/v2/home/accounts) and find the OU where you want to deploy the role.
+
+![Organization Home](./img/find_the_ou.png)
+
+The OU is that cryptic string underneat the name of the OU. In our example we want to deploy the role to all accounts in the organization, so we will use the Root OU. Which in this case is `r-7h7x`.
+
+#### Deploying the StackSet
+
+If you are not already in the correct account, log into the account where you want to deploy the StackSet. This account will be the account that manages the StackSet and the role that is deployed to all accounts in the organization unit.
+
+Navigate to the [Cloudformation Console](https://console.aws.amazon.com/cloudformation/home) and click on `Create Stack`. Leave all the default settings, enter the before mentioned S3 URL `https://resotopublic.s3.amazonaws.com/cf/resoto-stackset.template` in the `Amazon S3 URL` field and click `Next`.
+
+![Create Stack](./img/create_the_stack.png)
+
+On the next page, enter a Stack Name like `ResotoAccess`. In the Parameters section there are four parameters of which you need to fill out the first two. Numbers 3 and 4 are optional depending on your organization setup.
+
+1. DeploymentTargetOrganizationalUnitIds - This is the OU (or multiple OUs separated with a space) where you want to deploy the role. In our example we want to deploy the role to all accounts in the organization, so we will use the Root OU. Which in this case is `r-7h7x`.
+2. ResotoAccountID - This is the account ID of the account where you have Resoto deployed. This account will be the one the Role has a trust relationship with.
+3. StackSetCallAs - Defaults to DELEGATED_ADMIN but can be changed to SELF depending on the permissions you have in your organization and whether the account you are deploying the StackSet in is registered as a delegated administrator for your organization (recommended).
+4. StackSetPermissionModel - defaults to SERVICE_MANAGED but can be changed to SELF_MANAGED if you want to manage the StackSet permissions yourself. We prefer SERVICE_MANAGED as it allows us to turn on automatic deployments to accounts that we add to our organization in the future.
+
+![Stack Configuration](./img/configure_the_stack.png)
+
+Click `Next` on this as well as the next page and then click `Create Stack`.
```

Comment:
It is stupid - but maybe just add 2 screenshots and say click next. 
I would feel very comfortable seeing exactly the same content while I progress on this howto.